### ISSUE TITLE
Allow AssumeRoleWithWebIdentity to use Access token instead of OIDC ID token

### ISSUE THREAD

**Issue by chenkins** [iterate-ch/cyberduck#15182]
**Is your feature request related to a problem? Please describe.**
STS allows the usage of access tokens instead of OIDC ID token: https://docs.aws.amazon.com/STS/latest/APIReference/API_AssumeRoleWithWebIdentity.html

Extracted from #15177 / #15178 . See comment
https://github.com/iterate-ch/cyberduck/pull/15178#discussion_r1347604986

**Describe the solution you'd like**
Add property `s3.assumerole.tokentype`:
* if empty or set  to `id_token`, use ID token as implemented so far
* if set to `access_token`, use access token instead.

**Comment by chenkins** [comment]
Patch:
```
        final String webIdentityToken;
        // Use OIDC ID token by default. However, OAuth 2.0 access token may also be used:
        //  see https://docs.aws.amazon.com/STS/latest/APIReference/API_AssumeRoleWithWebIdentity.html
        if(StringUtils.isBlank(preferences.getProperty("s3.assumerole.tokentype")) || preferences.getProperty("s3.assumerole.tokentype").equals("id_token")) {
            webIdentityToken = oauth.getIdToken();
            if(log.isDebugEnabled()) {
                log.debug(String.format("AssumeRoleWithWebIdentity using OIDC ID token for %s", bookmark));
            }
        }
        else {
            webIdentityToken = oauth.getAccessToken();
            if(log.isDebugEnabled()) {
                log.debug(String.format("AssumeRoleWithWebIdentity using OAuth 2.0 access token for %s", bookmark));
            }
        }

        request.setWebIdentityToken(webIdentityToken);
```

**Comment by dkocher** [comment]
Making sure `openid` scope is requested from the OIDC provider will return an OIDC Id Token.

**Comment by chenkins** [comment]
Implemented by #15375-
